Sources/Features/Breadcrumb/BacktraceBreadcrumbFileHelper.swift

```diff
@@ -1,30 +1,26 @@
 import Foundation
-import Cassette
 
 enum BacktraceBreadcrumbFileHelperError: Error {
     case invalidFormat
 }
 
 @objc class BacktraceBreadcrumbFileHelper: NSObject {
 
-    /*
-     The underlying library CASQueueFile assigns a minimum of 4k (filled with zeroes).
-     Since we know that space will be allocated (and uploaded) anyways, set it as the minimum.
-     */
     private static let minimumQueueFileSizeBytes = 4096
 
     private let maximumIndividualBreadcrumbSize: Int
     private let maxQueueFileSizeBytes: Int
-    private let queue: CASQueueFile
+    
+    private let queue: Queue<Any>
+    private let breadcrumbLogURL: URL
 
-    /** CASQueueFile is not thread safe, so all interactions with it should be done synchronously through this DispathQueue */
     private let dispatchQueue = DispatchQueue(label: "io.backtrace.BacktraceBreadcrumbFileHelper@\(UUID().uuidString)")
 
     public init(_ breadcrumbSettings: BacktraceBreadcrumbSettings) throws {
-        self.queue = try CASQueueFile.init(path: breadcrumbSettings.getBreadcrumbLogPath().path)
-
+        
+        self.breadcrumbLogURL = try breadcrumbSettings.getBreadcrumbLogPath()
```

Comment:
why it can throw? Maybe it would be better to throw an error faster and here pass only data needed by BacktraceBreadcrumbFileHelper?